packages/apps-config/src/endpoints/productionRelayPolkadot.ts

```diff
@@ -127,7 +127,7 @@ export const prodParasPolkadot: Omit<EndpointOption, 'teleport'>[] = [
     info: 'centrifuge',
     paraId: 2031,
     providers: {
-      // Centrifuge: 'wss://fullnode.parachain.centrifuge.io', // https://github.com/polkadot-js/apps/issues/9366
+      Centrifuge: 'wss://fullnode.centrifuge.io',
```

Comment:
This cannot be the same as the "production" RPC.

An endpoint is either -

- connected to a relay (i.e. as a parachain)
- operating on it's own (i.e. not a parachain)

The two are mutually exlusive.

Reply:
Thanks for the clarification, fixed in c6889fc!